### PR TITLE
Added message to victoryscreen and defeatscreen if the player achieved a

### PR DESCRIPTION
**Pull request by justino10950**
highscore.

### CODE REVIEW

File: bust-a-move-framework/src/main/java/bustamove/gamestate/DefeatScreen.java [hunk 1]
```diff
@@ -89,8 +126,12 @@ public final void init(final GameContainer game,
         player2name = new Text("Player:", GameConfig.SIXT_LINE);
         player2name.centerText(game);
         player2score = new Text("Score:", GameConfig.SEVENTH_LINE);
-        player2score.centerText(game);
-        Button restart = new Button("Restart", GameConfig.FOURTH_LINE,
+        player2score.centerText(game);
+        player1highscore = new Text("", GameConfig.EIGTH_LINE);
```
Comment (peace-4-you) [inline bust-a-move-framework/src/main/java/bustamove/gamestate/DefeatScreen.java]: Why empty spaces ?

File: bust-a-move-framework/src/main/java/bustamove/gamestate/VictoryScreen.java [hunk 2]
```diff
@@ -87,9 +124,14 @@ public final void init(final GameContainer game,
         player1score.centerText(game);
         player2name = new Text("Player:", GameConfig.SIXT_LINE);
         player2name.centerText(game);
-        player2score = new Text("Score:", GameConfig.SEVENTH_LINE);
+        player2score = new Text("Score:", GameConfig.SEVENTH_LINE);
+        player2score.centerText(game);
+        player1highscore = new Text("", GameConfig.EIGTH_LINE);
```
Comment (peace-4-you) [inline bust-a-move-framework/src/main/java/bustamove/gamestate/VictoryScreen.java]: *

File: bust-a-move-framework/src/test/java/bustamove/game/HighscoreTest.java [hunk 3]
```diff
@@ -85,8 +85,9 @@ public void testHighscoreAddEntryAbove() {
 		int score2 = 100;
 		String name1 = "OP";
 		String name2 = "NOOB";
-		hs.addEntry(name2, score2);
-		hs.addEntry(name1, score1);
+		boolean b;
```
Comment (peace-4-you) [inline bust-a-move-framework/src/test/java/bustamove/game/HighscoreTest.java]: Unused variable

File: bust-a-move-framework/src/test/java/bustamove/game/HighscoreTest.java [hunk 4]
```diff
@@ -97,7 +98,7 @@ public void testHighscoreAddEntryAbove() {
 	@Test
 	public void testHighscoreSaveAndLoad() {
 		hs.load();
-		hs.addEntryAndSave("OP", 1000);
+		boolean b = hs.addEntryAndSave("OP", 1000);
```
Comment (peace-4-you) [inline bust-a-move-framework/src/test/java/bustamove/game/HighscoreTest.java]: Unused variable

**Comment by peace-4-you** [comment]
Also when you create an PR, please add in the description what and why you changed something.

File: bust-a-move-framework/src/main/java/bustamove/gamestate/DefeatScreen.java (hunk 1 above)
Comment (justino10950) [inline bust-a-move-framework/src/main/java/bustamove/gamestate/DefeatScreen.java]: Initializing the text at the correct position, the text is later filled in with the score the player has.

File: bust-a-move-framework/src/test/java/bustamove/game/HighscoreTest.java (hunk 3 above)
Comment (justino10950) [inline bust-a-move-framework/src/test/java/bustamove/game/HighscoreTest.java]: I will remove these.

**Pull request closed without merging**